static/main_menu.sh

```diff
@@ -27,19 +27,19 @@ case "$choice" in
     "Additional Apps")
         if network_ok
         then
-            run_static_script apps
+            run_script STATIC apps
         fi
     ;;
     "Nextcloud Configuration")
         if network_ok
         then
-            run_static_script nextcloud_configuration
+            run_script STATIC configuration
         fi
     ;;
     "Server Configuration")
         if network_ok
         then
-            run_static_script server_configuration
+            run_script STATIC server_configuration
```

Comment:
Can we please revert all changes in this commit?
As I said it will complicate things to fix #11 by a lot!

Reply:
I've reverted that.